tests/scripts/all.sh

```diff
@@ -715,6 +715,12 @@ component_check_test_cases () {
     unset opt
 }
 
+component_check_verify_psa_macros () {
+    msg "Check: verify PSA Crypto macros against spec" # ~ 2s
+    make -C programs test/verify_psa_macros
+    if_build_succeeded programs/test/verify_psa_macros
```

Comment:
This is independent of the library configuration, so I think it should be part of an existing component, like `tests/scripts/test_psa_constant_names.py`.